client/trigger_dockerfile_lint.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python
+
+import beanstalkc
+import json
+import os
+import sys
+
+if __name__ == "__main__":
+    bs = beanstalkc.Connection(host="openshift")
+    bs.use("master_tube")
+
+    name_space = sys.argv[1]
+    git_path = sys.argv[2]
+    filename = sys.argv[3]
+    notify_email = sys.argv[4]
+    job_name = sys.argv[5]
+
+    # having '/' in a value used in os.path.join generates unexpected paths
+    if git_path.startswith("/"):
+        git_path = git_path[1:]
+
+    if filename.startswith("/"):
+        filename = filename[1:]
+
+    lint_job_data = {}
+
+    try:
+        dockerfile_location = \
+            os.path.join(os.environ['DOCKERFILE_DIR'], git_path, filename)
+        with open(dockerfile_location) as f:
+            dockerfile = f.read()
+    except IOError:
+        print "==> Error opening the file %s" % dockerfile_location
+        print "==> Sending Dockerfile linter failure email"
+        response = {
+            "linter_results": True,
+            "action": "notify_user",
+            "namespace": name_space,
+            "notify_email": notify_email,
+            "job_name": job_name,
+            "msg": "Couldn't find the Dockerfile"
+        }
+
+        bs.use("master_tube")
+        jid = bs.put(json.dumps(response))
+        print "==>Put job on 'master_tube' tube with id: %d" % jid
+    else:
+        lint_job_data["name_space"] = name_space
```

Comment:
inconsistency in using the key `namespace` for job data.
Earlier we used `namespace`, here we are using `name_space`. This can cause the key error.